doc/source/whatsnew/v1.4.0.rst

```diff
@@ -15,10 +15,53 @@ including other versions of pandas.
 Enhancements
 ~~~~~~~~~~~~
 
-.. _whatsnew_140.enhancements.enhancement1:
+.. _whatsnew_140.enhancements.numeric_index:
 
-enhancement1
-^^^^^^^^^^^^
+More flexible numeric dtypes for indexes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Until now, it has only been possible to create numeric indexes with int64/float64/uint64 dtypes.
+It is now possible to create an index of any numpy int/uint/float dtype using the new :class:`NumericIndex` index type (:issue:`41153`):
+
+.. ipython:: python
+
+    pd.NumericIndex([1, 2, 3], dtype="int8")
+    pd.NumericIndex([1, 2, 3], dtype="uint32")
+    pd.NumericIndex([1, 2, 3], dtype="float32")
+
+In order to maintain backwards compatibility, calls to the base :class:`Index` will in
+pandas 1.x. return :class:`Int64Index`, :class:`UInt64Index` and :class:`Float64Index`.
+For example, the code below returns an ``Int64Index`` with dtype ``int64``:
+
+.. code-block:: ipython
```

Comment:
can this be an ipython example?

Reply:
This was done in anticipation of the removal of Int64Index etc. in 2.0, i.e. so this will not fail in 2.0.